content/author/manjot-sidhu.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Manjot Singh Sidhu"
+image = "/images/authors/mss.jpg"
+description = "Full Stack Developer"
++++
+
+{{< rawhtml >}}
+
+<p align="center">
+  <code>
+    <a href="mailto:manjot.techie@gmail.com" title="Gmail"><img width="22" alt="Gmail" src="https://github.com/manjotsidhu/manjotsidhu/raw/master/icons/Gmail.png"> Gmail</a></code>
```

Comment:
Alignment is a bit messed up here, please check!

![Screenshot 2021-07-09 at 3 10 45 PM](https://user-images.githubusercontent.com/45434030/125058397-dadd9300-e0c7-11eb-9bd2-7b68bfdf84b3.png)

Reply:
Hmm, consider it as a floating style icon 🤣. It's the same in my GitHub profile.
![image](https://user-images.githubusercontent.com/22657113/125058930-60f9d980-e0c8-11eb-9c7e-c36f969df07b.png)

Reply:
I am lazy to fix that